### PR TITLE
*: Remove `inject_connected` / `inject_disconnected` from docs

### PR DESCRIPTION
**Pull request by K0UR05H**
# Description
Both `inject_connected` and `inject_disconnected` are removed in dc8433e3.

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: swarm/src/behaviour.rs [hunk 1]
```diff
@@ -230,7 +230,7 @@ pub trait NetworkBehaviour: 'static {
     /// Informs the behaviour about an event generated by the handler dedicated to the peer identified by `peer_id`.
     /// for the behaviour.
     ///
-    /// The `peer_id` is guaranteed to be in a connected state. In other words, `inject_connected`
+    /// The `peer_id` is guaranteed to be in a connected state. In other words, `inject_connection_established`
```
Comment (thomaseizinger) [inline swarm/src/behaviour.rs]: Lets change this and other rustdoc appearances to an actual reference with `[``]`.

That will yield a compile-error if we ever hit this again.

**Review by thomaseizinger — COMMENTED** [review]
Thank you!

One suggestion.

File: swarm/src/behaviour.rs (hunk 1 above)
Comment (K0UR05H) [reply swarm/src/behaviour.rs]: Thanks for your review. I changed them.

**Review by mxinden — COMMENTED** [review]
Thanks for the help here. Please wrap the lines at 80 or 120 characters. Otherwise, this looks good to me.

**Review by mxinden — APPROVED** [review]
:pray: Thank you for the help!

**Pull request merged**